docs/build/Introduction/astar_family.md

```diff
@@ -34,28 +49,34 @@ The Shibuya native token symbol is SBY.
 
 To obtain test tokens from the faucet, please visit the Astar Portal and connect to Shibuya. If for any reason the faucet is empty, please contact the Astar team on Discord.
 
-### Rocstar
+### zKatana (Sepolia)
+
+An early iteration of the zkEVM testnet that used zk rollup architecture. Now deprecated.
 
-Rococo is a test Relay Chain used by the Polkadot & Kusama communities. Astar team has deployed a parachain to it called Rocstar, which is mainly used for cross-chain integrations with other teams in the ecosystem. To obtain test tokens for Rocstar, please contact Astar team on Discord.
+### zKyoto (Sepolia)
 
-The Rocstar native token symbol is ROC.
+zKyoto is a Validium-based network connected to the Ethereum Sepolia testnet and Polygon AggLayer. It is the testing and proving ground for dApps deployed on the Astar zkEVM.
 
 ## Mainnets
 
-Astar has two mainnets, like most parachains in the Polkadot ecosystem. One on Kusama Relay Chain, and the other on Polkadot Relay chain.
+Astar has three mainnets. One parachain on Kusama, One parachain on Polkadot, and a zkEVM-based Validium network on Ethereum Layer 2.
```

Comment:
I suggest we change the ordering of the mainnets intro, to 1. Astar zkEVM 2. Astar Parachain 3. Shiden Parachain